presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -67,6 +68,7 @@
     public static final String QUERY_MAX_RUN_TIME = "query_max_run_time";
     public static final String RESOURCE_OVERCOMMIT = "resource_overcommit";
     public static final String QUERY_MAX_CPU_TIME = "query_max_cpu_time";
+    public static final String QUERY_MAX_SCAN_BYTES = "query_max_scan_bytes";
```

Comment:
I think this name is a little confusing with the change in https://github.com/prestosql/presto/pull/2083/files (`query_max_input_data_size`). How about `query_max_scan_raw_bytes`? Or `query_max_raw_input_size` as the method name for stage stats suggests.

Reply:
> Could you also add a test in `TestQueryManager` as #2083 does?

Thanks for you reply, I will add a testcase and fix the comments.